website/docs/language/moved.mdx

```diff
@@ -16,8 +16,8 @@ The `moved` block programmatically changes the address of a resource. Refer to [
 The following list outlines field hierarchy, language-specific data types, and requirements in the `moved` block. 
 
 - [`moved`](#moved): map
-  - [`from`](#moved): string
-  - [`to`](#moved): string
+  - [`from`](#moved): instance address
+  - [`to`](#moved): instance address
```

Comment:
Targets are often groups of instances, so I think just "reference" here would be the correct type for documentation purposes.

Reply:
Done!